pkg/cache/azure_cache.go

```diff
@@ -189,9 +189,16 @@ func (t *TimedCache) Set(key string, data interface{}) {
 
 // Update updates the data cache for the key.
 func (t *TimedCache) Update(key string, data interface{}) {
-	_ = t.Store.Update(&AzureCacheEntry{
```

Comment:
Could you add some unit tests to catch this condition?

Reply:
Done.